configure.ac

```diff
@@ -746,8 +753,19 @@ if test x$use_pkgconfig = xyes; then
   m4_ifdef(
     [PKG_CHECK_MODULES],
     [
+      ICU_CPPFLAGS="-I$ICU_PREFIX/include"
+      CPPFLAGS="$ICU_CPPFLAGS $CPPFLAGS"
+      ICU_LIBS="-L$ICU_PREFIX/lib -licui18n -licuuc -licudata -dl"
+      AC_MSG_WARN([Using ICU_CPPFLAGS $ICU_CPPFLAGS])
+      AC_MSG_WARN([Using CPPFLAGS $CPPFLAGS])
+
+      AC_CHECK_HEADER([unicode/errorcode.h],,AC_MSG_ERROR(libicu headers missing))
+      AC_CHECK_LIB([icudata], [main], ICU_LIBS="-L$ICU_PREFIX/lib -licui18n -licuuc -licudata -dl",
+        AC_CHECK_LIB([icu18n], [main],ICU_LIBS=$ICU_LIBS, ICU_LIBS="-L$ICU_PREFIX/lib -lsicudt -lsicuin -lsicuio -lsicule -lsiculx -lsicutest -lsicutu -lsicuuc"))
+
```

Comment:
I investigate why configure without --with-icu fails, this section is for pkg-config you can replace all lines 756-765 with
```
PKG_CHECK_MODULES([ICUI18N], [icu-i18n],, [AC_MSG_ERROR(icu-i18n  not found.)])
```
Then it will make all needed flags and libs for compilation, `else` section (792-802) looks correct.

Reply:
I just tested this and still got the same error as before: `cannot determine absolute directory name of 'auto/lib'`. I modified configure.ac as suggested above, then reran autogen.sh and configure (`CXXFLAGS="-g" ./configure --enable-cxx --enable-static --disable-shared --with-pic --with-gui=no`) and make.

Reply:
That's the diff, it works here.
```
diff --git a/configure.ac b/configure.ac
index 3d9f513d9..243edf5b5 100644
--- a/configure.ac
+++ b/configure.ac
@@ -753,15 +753,9 @@ if test x$use_pkgconfig = xyes; then
   m4_ifdef(
     [PKG_CHECK_MODULES],
     [
-      ICU_CPPFLAGS="-I$ICU_PREFIX/include"
-      CPPFLAGS="$ICU_CPPFLAGS $CPPFLAGS"
-      ICU_LIBS="-L$ICU_PREFIX/lib -licui18n -licuuc -licudata -dl"
-      AC_MSG_WARN([Using ICU_CPPFLAGS $ICU_CPPFLAGS])
-      AC_MSG_WARN([Using CPPFLAGS $CPPFLAGS])
+      PKG_CHECK_MODULES([ICUI18N], [icu-i18n],, [AC_MSG_ERROR(icu-i18n  not found.)])
 
       AC_CHECK_HEADER([unicode/errorcode.h],,AC_MSG_ERROR(libicu headers missing))
-      AC_CHECK_LIB([icudata], [main], ICU_LIBS="-L$ICU_PREFIX/lib -licui18n -licuuc -licudata -dl",
-        AC_CHECK_LIB([icu18n], [main],ICU_LIBS=$ICU_LIBS, ICU_LIBS="-L$ICU_PREFIX/lib -lsicudt -lsicuin -lsicuio -lsicule -lsiculx -lsicutest -lsicutu -lsicuuc"))
 
       PKG_CHECK_MODULES([SSL], [libssl],, [AC_MSG_ERROR(openssl  not found.)])
       PKG_CHECK_MODULES([CRYPTO], [libcrypto],,[AC_MSG_ERROR(libcrypto  not found.)])
```